frontend/lib/util/nycha.tsx

```diff
@@ -26,4 +26,5 @@ export function isUserNycha(session: AllSessionInfo): boolean {
   return session.onboardingInfo
     ? session.onboardingInfo.leaseType === LeaseType.NYCHA
     : false;
+  // This is the only place
```

Comment:
I guess this will be removed before merging?

Reply:
Hey hey, just a reminder to maybe remove this before merging?  Unless there is a reason to keep it around?

Reply:
ah gosh, thank you! i gotta get my git lens set up so that if i commit one of these things it'll show it to me in my editor